generator/src/templates/example.ecr

```diff
@@ -3,10 +3,10 @@ require "../src/*"
 
 describe <%= "#{describe_name.inspect} do" -%>
 <% test_cases.each_with_index do |test_case, index| %>
-  <%= test_case.pending?(index) %> "<%= test_case.test_name %>" do <% -%>
-    <% test_case.workload.split("\n").each do |line| %>
+  <%= test_case.pending?(index) %> "<%= test_case.test_name %>" do<%- -%>
+    <%- test_case.workload.split("\n").each do |line| %>
     <%= line -%>
-    <% end %>
+    <%- end %>
```

Comment:
Using a dash inside `<...>` either eliminates previous indentation or removes the next newline:

    <%- ... %>:  removes previous indentation
    <% ... -%>:  removes next newline
    <%-= ... %>: removes previous indentation
    <%= ... -%>: removes next newline

Source: https://crystal-lang.org/api/1.4.1/ECR.html

Reply:
Interesting, I haven't played with templating in Crystal yet. Looks like eRuby mixed with some niceties from Jinja.